### PR TITLE
First comment issue fix

### PR DESCRIPTION
**Pull request by amassarwi**
While investigating #1601 I've found another issue:
Submitting new comment on a post with no comments didn't work.

I've added a small fix.

### CODE REVIEW

File: src/containers/Comments/Comment/CommentWrapper.jsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ const CommentWrapper = ({ comment, children, numOfReplies, ...props }) => {
   }, [showReplies]);
 
   const handleSubmitComment = useCallback(
-    async comment => {
+    comment => {
```
Comment (amassarwi) [inline src/containers/Comments/Comment/CommentWrapper.jsx]: @fernandoabolafio 
Could this be the cause of #1601 ? :open_mouth: :open_mouth:

Comment (fernandoabolafio) [reply src/containers/Comments/Comment/CommentWrapper.jsx]: I don't see why it would be the cause. Did you notice anything different after your change?

Comment (amassarwi) [reply src/containers/Comments/Comment/CommentWrapper.jsx]: I couldn't reproduce the issue locally, so not sure if it changed anything in the context of 1601

Comment (fernandoabolafio) [reply src/containers/Comments/Comment/CommentWrapper.jsx]: right, so why this change is needed then?

Comment (amassarwi) [reply src/containers/Comments/Comment/CommentWrapper.jsx]: it's an async function without any await calls & linter complained about it.

**Pull request merged**